themes/startwords/layouts/_default/list.atom.xml

```diff
@@ -44,6 +47,7 @@
             {{- end -}}
         {{- end }}
     {{- end }}
+    {{/* to here... */}}
```

Comment:
If you'd like to run locally immediately, remove these two sections.

Reply:
The error occurs because of `index site.Data.i18n.languages .Lang`. I assume that it's no longer structured as `site.Data.i18n.languages` anymore.